helper/schema/set_test.go

```diff
@@ -108,6 +108,13 @@ func TestSetUnion(t *testing.T) {
 	}
 }
 
+func TestSetLenNil(t *testing.T) {
+	var nilSet *Set
+	if nilSet.Len() != 0 {
+		t.Fatalf("Nil schema.Set should have 0 Len()")
+	}
+}
```

Comment:
I hit this before dropping in `ValueOrZero` to the `ResourceConfigData.Get()` function, but figured it was nice enough to leave in so code like this works:

```go
var s *schema.Set
if condition {
 s = realThing()
}
if s.Len() > 0 {
  handle(s)
}
```